packages/worker/src/utils.ts

```diff
@@ -28,3 +29,16 @@ export function convertToParentWorkflowType(
     namespace: parent.namespace,
   };
 }
+
+export function convertDeploymentVersion(
+  v: coresdk.common.IWorkerDeploymentVersion | null | undefined
+): WorkerDeploymentVersion | undefined {
+  if (v == null) {
+    return undefined;
+  }
+
+  return {
+    buildId: v.buildId ?? '',
```

Comment:
Is it really ok that we just default to empty strings here? That means `toCanonicalString` will resolve this to `'.'`.

Should we not instead make this default to the same as if no dployment version was given? i.e. return `undefined`?

Reply:
yeah, good point, Build ID at least will be set here if it's set at all